File: tldjfj123/Week07/삼총사/Solution.py
from itertools import combinations

def solution(number):
    combi = list(combinations(number, 3))
    res = 0

    for c in combi :
        if sum(c) == 0 :
            res += 1
    
    return res

solution([-2, 3, 0, 2, -5])

File: tldjfj123/Week07/삼총사/Solution.java
package 삼총사;

class Solution {
    public int solution(int[] number) {
        int res = 0;
        
        for (int i = 0; i < number.length; i++) {
            for (int j = i + 1; j < number.length; j++) {
                for (int k = j + 1; k < number.length; k++) {
                    if (number[i] + number[j] + number[k] == 0) {
                        res++;
                    }
                }
            }
        }
        
        return res;
    }
}